### DISCUSSION TITLE
Add new 150% and 200% thrall timers to "Timers" plugin

### DISCUSSION THREAD

**Discussion by big-bellend** [runelite/runelite#15985] [Ideas]
The new master and grandmaster combat achievement rewards give an 150% and 200% increase in duration to thralls, Would it be possible to add a toggle to the timers plugin to let you choose which reward you have (if any) or perhaps a check to automatically do this based on your CA completion. The current timers are wrong once you turn in the completion of the master/grandmaster tier to Ghommal.

**Comment by Adam-** [comment]
added in db21078f100738fb4989c45cf8f3aa22c9f8bb12
